apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -208,11 +208,11 @@ function extractFromCompletion(
  * Returns the AI question generation cache used for rate limiting.
  */
 let aiQuestionGenerationCache: Cache | undefined;
-export function getAiQuestionGenerationCache() {
+export async function getAiQuestionGenerationCache() {
```

Comment:
Reviewing this led me to open https://github.com/PrairieLearn/PrairieLearn/pull/12668.